demos/react/froala3/README.md

```diff
@@ -70,7 +70,7 @@ You will also see any lint errors in the console.
     // Define the toolbar content and the editor configuration.
     const toolbar = ['wirisEditor', 'wirisChemistry'];
     const froalaConfig = {
-        iframe: true,
+        // iframe: true,
```

Comment:
Since these are instructions inside a README file, I would refrain to include commented lines.

At the same time, it's not clear if the iframe value needs to be set to its default value.
Is it different from 'true'?

And I'm curious if this change is responsible for the fix.

Reply:
The change is not responsible for the fix, i forgot to comment the line.